File: components_page/components/button/outline.R
library(dashBootstrapComponents)
library(dashHtmlComponents)

buttons <- htmlDiv(
  list(
    dbcButton("Primary", outline = TRUE, color = "primary", class_name = "me-1"),
    dbcButton(
      "Secondary",
      outline = TRUE, color = "secondary", class_name = "me-1"
    ),
    dbcButton("Success", outline = TRUE, color = "success", class_name = "me-1"),
    dbcButton("Warning", outline = TRUE, color = "warning", class_name = "me-1"),
    dbcButton("Danger", outline = TRUE, color = "danger", class_name = "me-1"),
    dbcButton("Info", outline = TRUE, color = "info", class_name = "me-1"),
    dbcButton("Light", outline = TRUE, color = "light", class_name = "me-1"),
    dbcButton("Dark", outline = TRUE, color = "dark")
  )
)

File: components_page/components/button/outline.py
import dash_bootstrap_components as dbc
from dash import html

buttons = html.Div(
    [
        dbc.Button("Primary", outline=True, color="primary", className="me-1"),
        dbc.Button(
            "Secondary", outline=True, color="secondary", className="me-1"
        ),
        dbc.Button("Success", outline=True, color="success", className="me-1"),
        dbc.Button("Warning", outline=True, color="warning", className="me-1"),
        dbc.Button("Danger", outline=True, color="danger", className="me-1"),
        dbc.Button("Info", outline=True, color="info", className="me-1"),
        dbc.Button("Light", outline=True, color="light", className="me-1"),
        dbc.Button("Dark", outline=True, color="dark"),
    ]
)
